src/hotspot/share/opto/superword.cpp

```diff
@@ -2817,7 +2817,7 @@ void VTransform::adjust_pre_loop_limit_to_align_main_loop_vectors() {
 
 #ifdef ASSERT
   if (_trace._align_vector) {
-    tty->print_cr("\nVLoopTransformGraph::adjust_pre_loop_limit_to_align_main_loop_vectors:");
+    tty->print_cr("\nVLoopTransform::adjust_pre_loop_limit_to_align_main_loop_vectors:");
```

Comment:
For some reason I suggested `VLoopTransform` before.
```suggestion
    tty->print_cr("\nVTransform::adjust_pre_loop_limit_to_align_main_loop_vectors:");
```